storagesite/settings.py

```diff
@@ -146,8 +146,8 @@
     )
 }
 
-EMAIL_USE_TLS = True
-EMAIL_HOST='smtp.gmail.com'
-EMAIL_PORT=587
-EMAIL_HOST_USER=config('EMAIL_HOST_USER')
-EMAIL_HOST_PASSWORD=config('EMAIL_HOST_PASSWORD')
\ No newline at end of file
+# EMAIL_USE_TLS = True
+# EMAIL_HOST='smtp.gmail.com'
+# EMAIL_PORT=587
+# EMAIL_HOST_USER=config('EMAIL_HOST_USER')
+# EMAIL_HOST_PASSWORD=config('EMAIL_HOST_PASSWORD')
```

Comment:
Remember to uncomment important code before pushing if you had commented it out.